adam/experiment/experiment_utils.py

```diff
@@ -12,13 +12,16 @@
     instantiate_subcurricula,
     M6_CURRICULUM_ALL_OBJECTS,
 )
+
+from adam.ontology.phase2_ontology import GAILA_PHASE_2_ONTOLOGY
 from adam.ontology import IS_SPEAKER, IS_ADDRESSEE, THING
 from adam.ontology.phase1_ontology import (
     INANIMATE_OBJECT,
     CAN_BE_SAT_ON_BY_PEOPLE,
     ANIMATE,
 )
-from adam.ontology.phase2_ontology import GAILA_PHASE_1_ONTOLOGY, GAILA_PHASE_2_ONTOLOGY
+from adam.ontology.phase2_ontology import GAILA_PHASE_1_ONTOLOGY
+
```

Comment:
This import looks wrong? How/Why are we importing the Phase 1 ontology from the phase 2 module?